<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboard</class>
 <widget class="QMainWindow" name="AdminDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>715</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>715</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>715</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Admin Dashboard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/Logo/Logo.ico</normaloff>:/Logo/Logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2c3e50;
    font-family: &quot;Segoe UI&quot;, sans-serif;
}

QLabel#titleLabel {
    color: white;
    font-size: 22px;
    font-weight: bold;
    padding: 10px;
    qproperty-alignment: 'AlignCenter';
}

QLabel#welcomeLabel {
    color: #ecf0f1;
    font-size: 16px;
    padding: 6px;
    border-radius: 8px;
    qproperty-alignment: 'AlignCenter';
}

QPushButton {
    background-color: #3498db;
    color: white;
    font-size: 16px;
    padding: 8px;
    border-radius: 10px;
    border: none;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton#logoutButton {
    background-color: #e74c3c;
}

QPushButton#logoutButton:hover {
    background-color: #c0392b;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::Shape::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="LL_3">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>-1</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
    font-size: 28px;
    font-weight: bold;
    color: rgb(0, 255, 234);
    padding: 10px 0;
    border-bottom: 2px solid #3498db;
    letter-spacing: 1px;
}
</string>
              </property>
              <property name="text">
               <string>Admin Dashboard</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="welcomeLabel">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="add_new_customer">
              <property name="text">
               <string>Add New Customer/Admin</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_bank_account">
              <property name="text">
               <string>Add Bank Account</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edit_customer_info">
              <property name="text">
               <string>Edit Customer Info</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_specific_customer">
              <property name="text">
               <string>View Specific Customer Account</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_specific_bank_accounts">
              <property name="text">
               <string>View Specific Bank Account</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_all_customers">
              <property name="text">
               <string>View All Customers</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_all_bank_accounts">
              <property name="text">
               <string>View All Bank Accounts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="view_other_admins">
              <property name="text">
               <string>View All Admins</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edit_your_info">
              <property name="text">
               <string>Edit Your Info</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="logoutButton">
              <property name="text">
               <string>Logout</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
